src/dbus_api/pool/mod.rs

```diff
@@ -407,6 +440,22 @@ pub fn get_pool_properties(
             consts::POOL_OVERPROV_PROP => shared::pool_overprov_enabled(pool),
             consts::POOL_NO_ALLOCABLE_SPACE_PROP => shared::pool_no_alloc_space(pool),
             consts::POOL_METADATA_VERSION_PROP => shared::pool_metadata_version(pool)
+        },
```

Comment:
I believe that you should remove POOL_METADATA_VERSION_PROP from the 3_7 interface.

Reply:
Okay, done.